app/poro/movie.rb

```diff
@@ -1,19 +1,46 @@
 class Movie
   attr_reader :tmdb_id,
               :title,
-              :vote_average
+              :vote_average,
+              :runtime,
+              :description
 
   def initialize(movie_params)
+    @movie_params = movie_params
     @tmdb_id = movie_params[:id]
     @title = movie_params[:title]
     @vote_average = movie_params[:vote_average]
+    @runtime = movie_params[:runtime]
+    @description = movie_params[:overview]
+  end
+
+  def genres
+    @movie_params[:genres].map do |genre|
+      genre[:name]
+    end
+  end
+
+  def cast_members
```

Comment:
This took care of our nil issue when consuming one end point that wouldn't have all the attributes if they were attr_readers

Reply:
That's right! Don't need a guard clause if the method is not invoked! --taps finger to temple--